libs/utils/utils.js

```diff
@@ -82,6 +82,8 @@ const AUTO_BLOCKS = [
 const ENVS = {
   local: {
     name: 'local',
+    ims: 'stg1',
+    adobeIO: 'cc-collab-stage.adobe.io',
```

Comment:
Can we also add `account: 'stage.account.adobe.com'` when we are on it? So that you can instantly log in with a stage account